app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -142,8 +147,9 @@ class Identification extends React.Component {
       }
     );
   };
-
-  getWorkflowValue = () => (this.state.current.isolation ? WORKFLOW_OPTIONS[1] : WORKFLOW_OPTIONS[0]);
+  // TODO: At the current moment, it is assumed that at least one of `isolation` and `exposure` will be an available workflow option.
```

Comment:
Remove keyword `TODO`
```suggestion
  // At the current moment, it is assumed that at least one of `isolation` and `exposure` will be an available workflow option.
```

Reply:
@DPC15038

Reply:
done.